core/jvm/src/main/scala/tf/bug/dirs/JVMTarget.scala

```diff
@@ -0,0 +1,16 @@
+package tf.bug.dirs
+
+
+
+object JVMTarget extends Target {
+  def platform: Platform = {
+    val osName = System.getProperty("os.name")
+    if (osName.contains("Windows")) {
+      Platform.Windows
+    } else {
```

Comment:
`contains("Mac") || contains("Darwin")`? Not sure if `darwin` will report lower-case here though...

Reply:
I got the "Mac" part by running`getProperty` locally.
AFAIK, `os.name` never returns "darwin". However only evidence I can find is are blogposts.